src/org/labkey/test/tests/UserClonePermissionTest.java

```diff
@@ -0,0 +1,179 @@
+package org.labkey.test.tests;
+
+import org.jetbrains.annotations.Nullable;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.categories.Daily;
+import org.labkey.test.pages.user.ClonePermissionsPage;
+import org.labkey.test.util.ApiPermissionsHelper;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.PermissionsHelper;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Category({Daily.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 3)
+public class UserClonePermissionTest extends BaseWebDriverTest
+{
+    private static final String CLONED_SOURCE_SITE_USER = "sourcesiteuser@clonepermission.test";
+    private static final String SOURCE_SITE_USERNAME = "sourcesiteuser";
+    private static final String CLONED_SOURCE_APP_USER = "sourceappuser@clonepermission.test";
+    private static final String CLONED_TARGET_SITE_USER = "targetsiteuser@clonepermission.test";
+    private static final String TARGET_SITE_USERNAME = "targetsiteuser";
+    private static final String CLONED_TARGET_APP_USER = "targetappuser@clonepermission.test";
+    private static final String TARGET_APP_USERNAME = "targetappuser";
```

Comment:
Don't hard-code the user display names. They might not match what's here (usually due to collisions with other tests). `_userHelper.getDisplayNameForEmail` will get them when they are needed.